src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/ElementEditor.tsx

```diff
@@ -248,6 +256,11 @@ const ElementEditor = ({
         type: ResourceActionType.MODIFY_BUNDLE_ENTRY,
         payload: bundleEntry,
       });
+      setToastType("success");
+      setToastMessage(
+        `${type} has successfully been applied to the test case. To save your changes please click 'Save'`
```

Comment:
Please add a period at the end of the message.

Reply:
@jkotanchik-SB Added, thanks Joe!